contrib/platform/src/com/sun/jna/platform/win32/COM/COMBindingBaseObject.java

```diff
@@ -92,7 +92,7 @@ public COMBindingBaseObject(String progId, boolean useActiveInstance,
         assert COMUtils.comIsInitialized() : "COM not initialized";
 
         CLSID.ByReference clsid = new CLSID.ByReference();
-        HRESULT hr = Ole32.INSTANCE.CLSIDFromProgID(progId, clsid);
+        HRESULT hr = Ole32.INSTANCE.CLSIDFromProgID(new WTypes.LPOLESTR(progId), clsid);
```

Comment:
Not needed, `String` is handled properly.

Reply:
According to MSDN, LPOLESTR is an alias for wchar_t*, thus it needs to be WString even on ANSI API?